YoloV8 serverless support

hardikdava opened the pull request:
### Motivation and context

Yolov8 object detection support added.

This pr is for #6471. 
 
### How has this been tested?

```
sh ./serverless/deploy_cpu.sh serverless/pytorch/ultralytics/yolov8
```
and
```
sh ./serverless/deploy_gpu.sh serverless/pytorch/ultralytics/yolov8
```


### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

### License

Please check license since `ultralytics` is released under different license and so is `yolov5`

nmanovic commented on CHANGELOG.md:
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## \[Unreleased]
 ### Added
 - Multi-line text attributes supported (<https://github.com/opencv/cvat/pull/6458>)
+- Added YoloV8 serverless support for semi/automatic annotation
```
```suggestion
- Added YoloV8 serverless support for semi/automatic annotation (<https://github.com/opencv/cvat/pull/6472>)
```

nmanovic commented on serverless/pytorch/ultralytics/yolov8/nuclio/main.py:
```diff
@@ -0,0 +1,58 @@
+import json
+import base64
+from PIL import Image
+import io
+import torch
```
```suggestion
import io
```

nmanovic commented on serverless/pytorch/ultralytics/yolov8/nuclio/main.py:
```diff
@@ -0,0 +1,57 @@
+
+    results = []
+    
+    if boxes.shape[0] > 0:
+
```
```suggestion
    if boxes.shape[0] > 0:
```

nmanovic commented:
@hardikdava , could you please update [‎site/content/en/docs/manual/advanced/ai-tools.md](https://github.com/opencv/cvat/blob/0715ec0483fa4a69e33c878119a84a7b4b88a8f6/site/content/en/docs/manual/advanced/ai-tools.md?plain=1#L206) as well? Please add information about YOLO v8

hardikdava commented:
> @hardikdava , could you please fix pylint warnings?
> 
> ![image](https://user-images.githubusercontent.com/40690625/253330255-0a2428b5-8b86-4fcc-9f02-d6e364ec2f40.png)
> 
> Also function-gpu.yml doesn't have `pip install supervision`. It leads to a problem.

Done. Added `pip install supervision`



> @hardikdava , could you please update [‎site/content/en/docs/manual/advanced/ai-tools.md](https://github.com/opencv/cvat/blob/0715ec0483fa4a69e33c878119a84a7b4b88a8f6/site/content/en/docs/manual/advanced/ai-tools.md?plain=1#L206) as well? Please add information about YOLO v8

Done. Added information about YoloV8 in ai-tools.md

hardikdava commented:
is there any update on the merging?

bsekachev commented:
Hi @hardikdava 

Thank you for the contribution. 
But, unfortunately we can't use YOLO v3/v5/v8 in our product, because recent change of their license to AGPL which is not compatible with MIT.
